apps/grant/src/index.js

```diff
@@ -1,10 +1,10 @@
+/* eslint-disable  */
+
 const express = require('express');
 const axios = require('axios');
 const app = express();
 
-const cloudhaven = 'http://127.0.0.1:5000/api';
-// reference
-// https://blog.logrocket.com/how-to-make-http-requests-like-a-pro-with-axios/
+const chURL = 'http://127.0.0.1:5000/api';
```

Comment:
This variable name isn't very meaningful (and it doesn't follow standard camel case formatting)

Reply:
Thanks for the catch, should be fixed now